### PR TITLE
feat: am facut 2 fisiere navigation si am creat liste #54

### PR DESCRIPTION
**Pull request by antonia2203**

### CODE REVIEW

File: src/antonia2203/navigation.pug [hunk 1]
```diff
@@ -0,0 +1,14 @@
+html 
+  head 
+    title Navigaion Tag 
+  body 
+    nav.menu
+      ul(style="list-style-type:circle")
+        li
+          a(href="/") Home
```
Comment (marianzburlea) [inline src/antonia2203/navigation.pug]: the anchor tags are missing the **title** attribute

**Review by MariaTeo — CHANGES_REQUESTED** [review]
missing title attribute for anchor tags

**Review by marianzburlea — CHANGES_REQUESTED** [review]

**Review by MariaTeo — APPROVED** [review]
Nice one

**Review by marianzburlea — CHANGES_REQUESTED** [review]
update closing tags

**Review by MariaTeo — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**